src/main/java/com/duke/task/TaskList.java

```diff
@@ -0,0 +1,35 @@
+package com.duke.task;
+
+import com.duke.task.Task;
+
+import java.util.ArrayList;
+
+public class TaskList {
+
+    private ArrayList<Task> list;
+    private int count;
+
+    public TaskList(ArrayList<Task> list) {
+        this.list = list;
+        this.count = list.size();
+    }
+
+    public void add(Task task) {
+        list.add(task);
+        count++;
+    }
+
+    public Task remove (int index) {
```

Comment:
```suggestion
    public Task remove(int index) {
```
suggest fixing spacing